pkg/virt-controller/services/template_test.go

```diff
@@ -56,6 +57,71 @@ var _ = Describe("Template", func() {
 				Expect(pod.Spec.Containers[0].Command).To(Equal([]string{"/virt-launcher", "-qemu-timeout", "60s"}))
 			})
 		})
+		Context("migration template with correct parameters", func() {
+			It("should work", func() {
+				vm := v1.VM{ObjectMeta: corev1.ObjectMeta{Name: "testvm", UID: "1234"}, Spec: v1.VMSpec{Domain: &v1.DomainSpec{}}}
+				node1 := corev1.Node{
+					Status: corev1.NodeStatus{
+						Addresses: []corev1.NodeAddress{
+							corev1.NodeAddress{
+								Type:    corev1.NodeExternalIP,
+								Address: "127.0.0.1",
+							},
+						},
+					},
+				}
+				node2 := corev1.Node{
+					Status: corev1.NodeStatus{
+						Addresses: []corev1.NodeAddress{
+							corev1.NodeAddress{
+								Type:    corev1.NodeExternalIP,
+								Address: "127.0.0.2",
+							},
+						},
+					},
+				}
+
+				job, err := svc.RenderMigrationJob(&vm, &node1, &node2)
+				Expect(err).To(BeNil())
+				Expect(job.Spec.Template.Spec.RestartPolicy).To(Equal(coreapi.RestartPolicyNever))
+			})
+		})
+		Context("migration template with incorrect parameters", func() {
+			It("should error on missing source address", func() {
+				vm := v1.VM{ObjectMeta: corev1.ObjectMeta{Name: "testvm", UID: "1234"}, Spec: v1.VMSpec{Domain: &v1.DomainSpec{}}}
+				node1 := corev1.Node{}
+				node2 := corev1.Node{
```

Comment:
Consider creating always two valid nodes in a BeforeEach() and explicitly remove the parts in the test which should cause the error.